app/controllers/sessions_controller.rb

```diff
@@ -4,11 +4,20 @@ class SessionsController < ApplicationController
   def create
     session[:user_token] = auth_hash.fetch(:credentials).fetch(:token)
 
-    redirect_to root_url
+    redirect_to redirect_url
   end
 
   private
 
+  def redirect_url
+    custody_suite_root_url if user_role == "cso"
+  end
+
+  # TODO: This is hacky
+  def user_role
+    @user_role ||= current_user.organisations.map { |o| o["roles"] }.flatten.uniq.first
```

Comment:
There's `current_user.roles` array now, which should be used. We only use roles from 1 organisation at the time.

Also I'm thinking that as we have a very similar code in `PolicyContext`, we should maybe put this on the current user, so at least it's in 1 place and when we change it we have it under control. 

If we don't consolidate, this should be: `current_user.roles.uniq.first`.